packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts

```diff
@@ -53,6 +55,10 @@ const checkboxStyles: ComponentSlotStylesInput<CheckboxProps & CheckboxState, Ch
     margin: v.margin,
     padding: v.padding,
 
+    [`&.${Checkbox.slotClassNames.indicator}`]: {
+      marginTop: pxToRem(2),
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/64005180-214b2980-cb10-11e9-8357-cc78b57a9e55.png)

![image](https://user-images.githubusercontent.com/14183168/64005130-04165b00-cb10-11e9-9e2c-7646d7c3df82.png)

This override is really strange. I expect that `margin` variable will be overridden instead:

#### `checkboxStyles.ts`
```diff
-    [`&.${Checkbox.slotClassNames.indicator}`]: {
-      marginTop: pxToRem(2),
-    },
```
#### `checkboxVariables.ts`
```diff
-  margin: `${pxToRem(20)}, 0`, // THERE IS A MISTAKE, COMMA!
+  margin: `pxToRem(2) 0 0 0`,
```

Reply:
Thank you for catching that. I wonder why that mistake was not causing any errors.